xen/arch/x86/boot/head.S

```diff
@@ -475,6 +475,15 @@ __pvh_start:
          *   which is supported by a given SINIT ACM
          */
 slaunch_stub_entry:
+#ifdef XEN_BUILD_EFI
+        mov     %ebx, %esi
+        sub     $sym_offs(slaunch_stub_entry), %esi
+        cmpb    $0, sym_esi(slaunch_efi_boot)
+        je      1f
+        jmp     slaunch_efi_jumpback
+1:
```

Comment:
I'm not sure if it is the case in Xen, but some reviewers are picky when it comes to local labels in conditionally compiled code.

```suggestion
        jne     slaunch_efi_jumpback
```

Reply:
https://github.com/TrenchBoot/xen/compare/f3db106684be8f16b54f75d1dccb23b1046c3d63..1278ae8b5f51de658b423849ccdfdb1a373acc8a